runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java

```diff
@@ -119,4 +119,35 @@ public class SentryConfig {
      */
     @ConfigItem
     public OptionalDouble tracesSampleRate;
+
+    /**
+     * When set to {@code true}, a proxy can be configured that should be used for outbound requests. This is also
+     * used for HTTPS requests.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean proxyEnabled;
```

Comment:
If you create a SentryProxyConfig, you can use: `@ConfigItem(name = ConfigItem.PARENT)` to enable it (which is a common practice in Quarkus extensions)